src/interactions/mie.jl

```diff
@@ -31,34 +31,14 @@ Mie(m, n) = Mie(m, n, false)
     σ = sqrt(atom_i.σ * atom_j.σ)
     ϵ = sqrt(atom_i.ϵ * atom_j.ϵ)
 
-    cutoff = inter.cutoff
     m = inter.m
     n = inter.n
     # Derivative obtained via wolfram
     const_mn = inter.mn_fac * ϵ
     σ_r = σ / r
     params = (m, n, σ_r, const_mn)
 
-    if cutoff_points(C) == 0
-        f = force_nocutoff(inter, r2, inv(r2), params)
-    elseif cutoff_points(C) == 1
-        sqdist_cutoff = cutoff.sqdist_cutoff * σ2
-        r2 > sqdist_cutoff && return zero(coord_i)
-
-        f = force_cutoff(cutoff, r2, inter, params)
-    elseif cutoff_points(C) == 2
-        sqdist_cutoff = cutoff.sqdist_cutoff * σ2
-        activation_dist = cutoff.activation_dist * σ2
-
-        r2 > sqdist_cutoff && return zero(coord_i)
-
-        if r2 < activation_dist
-            f = force_nocutoff(inter, r2, inv(r2), params)
-        else
-            f = force_cutoff(cutoff, r2, inter, params)
-        end
-    end
-
+    f = force_nocutoff(inter, r2, inv(r2), params)
```

Comment:
The Mie potential doesn't have cutoffs by definition?
I saw a cutoff in the original PR
https://github.com/longemen3000/Molly.jl/blob/79336da46c8271379a1240f1f2be54779084c332/src/forces.jl#L84
and that's why I added the cutoff part.

Reply:
Yeah just added it back.

Reply:
Yeah just added it back.